portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,140 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio HQ: Charity CJ Tarver</title>
+    <link rel="icon" 
+        type="image/png" 
+        href="https://media.istockphoto.com/vectors/wolf-icon-vector-id844157592?b=1&k=6&m=844157592&s=612x612&w=0&h=hQF8q7G3X6RC-qjtGolEwwZwdaeCfUfHsl3nlIhqixU="/>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+    <header>
+    <nav class = "nav-header">
+    <ul class="nav-links">
```

Comment:
The code is well formatted except this part. Consider indenting this section as well !